pkgs/applications/audio/openutau/update.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p bash curl nix jq common-updater-scripts
+
+set -euo pipefail
+
+nixpkgs="$(git rev-parse --show-toplevel || (printf 'Could not find root of nixpkgs repo\nAre we running from within the nixpkgs git repo?\n' >&2; exit 1))"
```

Comment:
Yeah it would work for any git repo this package lives in with a similar default.nix entrypoint. I actually use it from my own repo. But I just say "nixpkgs" to reduce confusion in these scripts (this is mostly copied from other packages I have that aren't yet supported by nix-update)